lib/arena.rb

```diff
@@ -1,2 +1,34 @@
+
+
 class Arena
+  attr_accessor :name, :gladiators
+  # @@count_glads = 0
```

Comment:
This wouldn't necessarily help in refactoring your code, but how could this commented-out class variable be used in your `Arena` methods?